src/test/resources/config.properties

```diff
@@ -0,0 +1 @@
+browser=chrome
```

Comment:
By putting this config file in git, it makes it hard for different users to use different configs. Anyone who tried to use a different browser would have to commit a change to the config. This kind of thing should be in either an ignored config file, or read via a command line argument

Reply:
Added to gitignore